### PR TITLE
refactor: Skills and agent moved to deprecated

### PR DESCRIPTION
**Pull request by IgnatovFedor**

### CODE REVIEW

File: docs/intro/overview.rst [hunk 1]
```diff
@@ -33,7 +31,7 @@ Key Concepts
    end-to-end mode being joined in a chain.
 -  ``Skill Manager`` performs selection of the ``Skill`` to generate
    response.
--  ``Chainer`` builds an agent/model pipeline from heterogeneous
+-  ``Chainer`` builds an model pipeline from heterogeneous
```
Comment (yoptar) [inline docs/intro/overview.rst]:
```suggestion
-  ``Chainer`` builds a model pipeline from heterogeneous
```

File: README.md [hunk 2]
```diff
@@ -216,6 +216,25 @@ and others in the Integrations section for more info.
...
+- server logs formatting was changed at [REST API](http://docs.deeppavlov.ai/en/0.7.0/integrations/rest_api.html),
+  [Yandex Alice](http://docs.deeppavlov.ai/en/0.7.0/integrations/yandex_alice.html),
+  [Amazon Alexa](http://docs.deeppavlov.ai/en/0.7.0/integrations/amazon_alexa.html) and
+  [Microsoft Bot Framework](http://docs.deeppavlov.ai/en/0.7.0/integrations/ms_bot.html) integrations
+- in dialog logger config file `dialog_logger_config.json` `agent_name` parameter was renamed to `logger_name`,
+  the default value was changed
+- [agent](http://docs.deeppavlov.ai/en/0.6.0/apiref/core/agent.html),
+  [skill](http://docs.deeppavlov.ai/en/0.6.0/apiref/core/skill.html), 
+  [eCommerce Bot](http://docs.deeppavlov.ai/en/0.6.0/features/skills/ecommerce.html), 
+  [Pattern Matching](http://docs.deeppavlov.ai/en/0.6.0/features/skills/pattern_matching.html) moved to `deeppavlov.deprecated`
```
Comment (yoptar) [inline README.md]: Maybe just list `Agent, Skill, eCommerce Bot and Pattern Matching classes were moved to [deeppavlov.deprecated](deeppavlov/deprecated/)`? Or something like that

File: README.md [hunk 3]
```diff
@@ -216,6 +216,25 @@ and others in the Integrations section for more info.
...
+- [AIML Skill](http://docs.deeppavlov.ai/en/0.7.0/features/skills/aiml_skill.html),
+  [RASA Skill](http://docs.deeppavlov.ai/en/0.7.0/features/skills/rasa_skill.html),
+  [Yandex Alice](http://docs.deeppavlov.ai/en/0.7.0/integrations/yandex_alice.html),
+  [Amazon Alexa](http://docs.deeppavlov.ai/en/0.7.0/integrations/amazon_alexa.html),
+  [Microsoft Bot Framework](http://docs.deeppavlov.ai/en/0.7.0/integrations/ms_bot.html) and
+  [Telegram integration](http://docs.deeppavlov.ai/en/0.7.0/integrations/telegram.html) interfaces were changed
+- `/start` and `/help` Telegram messages were moved from `models_info.json` to `connector_config.json` file
```
Comment (yoptar) [inline README.md]: Can we link the file(s)?

File: README.md (hunk 2 above)
Comment (IgnatovFedor) [inline README.md]:
```suggestion
Agent, Skill, eCommerce Bot and Pattern Matching classes were moved to [deeppavlov.deprecated](deeppavlov/deprecated/)
```

File: README.md (hunk 3 above)
Comment (IgnatovFedor) [inline README.md]:
```suggestion
- `/start` and `/help` Telegram messages were moved from `models_info.json` to [connector_config.json](/deeppavlov/utils/settings/connector_config.json)
```

**Pull request merged**